devel/zmq/Portfile

```diff
@@ -88,9 +100,7 @@ homepage            http://www.zeromq.org/
 
 dist_subdir         zmq
 
-# use autoconf / automake
-
-use_autoconf        yes
+#use_autoconf        yes
```

Comment:
Do the non-current ZMQ ports require autoconf at all now? You're moving the release to CMake ... what about devel? I'd guess it can use CMake if the release can! If so, then I'd say to remove the 'use_autoconf yes" from the above subports and instead add in a clause here to either use autoconf ("yes" along with the 2 lines below) or "PortGroup cmake 1.1" depending on the subport.

Reply:
Sure, the devel subport can also use cmake. I have no idea if any of the older subports support cmake.

Reply:
Yeah i don't really care about the older ZMQ ports ... not even sure they are used much, if at all. It'd be worth deprecating them if at all possible ... in another PR sometime ...